common/test/unit/com/thoughtworks/go/domain/materials/git/GitCommandTest.java

```diff
@@ -352,6 +403,41 @@ public void shouldReturnFalseIfTheGivenBranchDoesNotContainTheRevision() {
     }
 
     @Test
+    public void shouldReturnTrueIfVersionHigherThan1dot6Linux() throws Exception {
+        assertThat(git.isVersionEqualToOrHigherThan(GIT_VERSION_1_6_0_2, 1.6f), Is.is(true));
+    }
+
+    @Test
+    public void shouldReturnTrueIfVersionLowerThanExpectedLinux() throws Exception {
```

Comment:
You are asserting on false while saying shouldReturnTrue. Please change the name of the test to indicate what the test is checking for as far as possible.